showup/migrations/0001_initial.py

```diff
@@ -243,4 +243,12 @@ class Migration(migrations.Migration):
                 name="You can only swipe on another particular person for aparticular event once",
             ),
         ),
+        migrations.CreateModel(
+            name='Squad',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email_id', models.EmailField(max_length=254)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='squad', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
     ]
```

Comment:
You shouldn't be editing the initial migration. Any new changes should go in new migration files.